packages/tools/src/enums/AnnotationStyleStates.ts

```diff
@@ -11,12 +11,15 @@
  *   The annotation has been selected by the user;
  * Locked:
  *   The annotation has been locked;
+ * AutoGenerated:
+ *   The annotation has been auto generated;
  */
 enum AnnotationStyleStates {
   Default = '',
   Highlighted = 'Highlighted',
   Selected = 'Selected',
   Locked = 'Locked',
+  AutoGenerated = 'AutoGenerated',
```

Comment:
I'm not sure we end up using this, but I think it is a worthwhile change, so I'm going to leave it in.